### PR TITLE
130 update team details for mdma project

### LINKED ISSUE #130: update team details for MDMA project

### PR DESCRIPTION
**Pull request by Anjali0407-git**
Fixes #130

After updates:
image

### CODE REVIEW

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
I love the addition of a "portfolio" icon! great idea!

We need a label for the old team who are no longer active members. Perhaps "(proof of concept)"? That is accurate, but maybe too wordy. What do you think?

**Comment by kungfuchicken** [comment]
"POC" is sometimes used, but it als is used for "piece of crap" and I don't want to give that impression at all. If you're comfortable with it let's just go with "`(proof of concept)`" all lowercase to match the labels on other pages.

**Comment by Anjali0407-git** [comment]
Got it, I am adding that now

**Review by kungfuchicken — APPROVED** [review]
